src/Components/Card.tsx

```diff
@@ -51,6 +51,21 @@ const ServiceName = styled.p`
   padding: 0 1rem;
 `
 
+const ImageText = styled(ServiceName)`
```

Comment:
This can just be `styled.p` if you aren't using much from the ServiceName styles. It looks like they're mostly either overridden or won't apply here?